service-base/pom.xml

```diff
@@ -43,6 +43,11 @@
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-core</artifactId>
     </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-jsr310</artifactId>
```

Comment:
we'll need a CQ for this

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20419

Reply:
any news on this yet?

Reply:
the CQ is approved